internal/command/machine/start.go

```diff
@@ -38,31 +38,32 @@ func newStart() *cobra.Command {
 
 func runMachineStart(ctx context.Context) (err error) {
 	var (
-		out       = iostreams.FromContext(ctx).Out
-		appName   = app.NameFromContext(ctx)
-		machineID = flag.FirstArg(ctx)
+		out     = iostreams.FromContext(ctx).Out
+		appName = app.NameFromContext(ctx)
+		args    = flag.Args(ctx)
 	)
 
-	app, err := appFromMachineOrName(ctx, machineID, appName)
-	if err != nil {
-		return
-	}
+	for _, machineID := range args {
+		app, err := appFromMachineOrName(ctx, machineID, appName)
+		if err != nil {
+			return fmt.Errorf("could not make flaps client: %w", err)
+		}
 
-	flapsClient, err := flaps.New(ctx, app)
-	if err != nil {
-		return fmt.Errorf("could not make flaps client: %w", err)
-	}
+		flapsClient, err := flaps.New(ctx, app)
```

Comment:
Just a comment here. Having to create a new flaps client for every app shows to me that the flaps package needs a bit of refactoring, so you can set the app on the client instead of creating a new one each time. Not required here of course.

Reply:
Yeah I’ve been thinking about that too. It’s kinda of a blocker for my restart postgres work.

Reply:
In general we have a lot of overlap now across postgres and machine deployment. It feels like a new abstraction would be helpful, but not sure exactly what yet. One problem we have is that flaps is a per-org endpoint, so I believe we need a client per org right now.